partiql-ast/src/main/java/org/partiql/ast/v1/expr/ExprVar.java

```diff
@@ -0,0 +1,48 @@
+package org.partiql.ast.v1.expr;
+
+import org.jetbrains.annotations.NotNull;
+import org.partiql.ast.v1.AstNode;
+import org.partiql.ast.v1.AstVisitor;
+import org.partiql.ast.v1.Identifier;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * TODO docs, equals, hashcode
+ */
+public class ExprVar extends Expr {
```

Comment:
```suggestion
// or VarRef, VarReference, Ref, BindingRef, etc.
public class Reference extends Expr {
```

I like this modeling as it provides a distinction between `AS <identifier>` vs referencing fields/globals/etc. In SQL:1999, this would be a `<column reference>` or `<table reference>`, though, PartiQL is a bit nuanced, since we allow for both everywhere an expression is allowed.

Anyways, the use of `reference` aligns with SQL.

```
<select list> ::=
  <asterisk>
  | <select sublist> [ { <comma> <select sublist> }... ]

<select sublist> ::=
  <derived column>
  | ...

<derived column> ::=
  <value expression> [ <as clause> ]

<as clause> ::= [ AS ] <column name>

<nonparenthesized value expression primary> ::=
  <unsigned value specification>
  | <column reference>
  | ...

<column reference> ::=
  <basic identifier chain>
  | MODULE <period> <qualified identifier> <period> <column name>
```

Reply:
I'll change to `ExprVarRef` to be a bit more descriptive